### PR TITLE
Define "integer" to include zero.

### LINKED ISSUE #280: ddl.dic: `_type.contents` definition of an Integer excludes 0
The `_type.contents` data item describes the Integer type as:
```
positive or negative integer number
```
Since zero is neither positive nor negative (or both positive and negative), it is currently excluded from the allowed values.

### PR DESCRIPTION
**Pull request by rowlesmr**
Will close #280 

Previously, zero was excluded.

### CODE REVIEW

File: ddl.dic [hunk 1]
```diff
@@ -1729,7 +1729,7 @@ save_type.contents
 ;
          Integer
 ;
-         positive or negative integer number
+         An integer. Positive and negative integers, and zero, are included.
```
Comment (vaitkus) [inline ddl.dic:1732]:
```suggestion
         An integer. Belongs to a numerical set that consists of positive integers, negative integers and the number zero.
```

Comment (vaitkus) [reply ddl.dic:1732]: Simply proposed an alternative formulation of the same statement. Feel free to ignore if it seems too verbose.

Comment (rowlesmr) [reply ddl.dic:1732]: It does seem a little self-referential to define a thing in terms of itself.

[Wolfram|Alpha](https://www.wolframalpha.com/examples/mathematics/numbers/integers) define integers as:
> The set of integers consists of the counting numbers, their negations and zero.

Perhaps we could use:
> An integer. Integers consist of the positive and negative counting numbers, and zero.

Comment (vaitkus) [reply ddl.dic:1732]: The term "negative counting numbers" looks a bit off to me as counting numbers by definition should not be negative. Maybe something like "`A number from the set of all integers.`" would work? We could potentially further specify what makes up the integer set, but this seems like a well established mathematical term so there is probably no need to be too verbose. The important thing in my eyes is to specify two things:
- That the allowed values include a zero.
- That the `Integer` in question refers to the mathematical concept of an integer and not to a programmers view of an integer.

Both things seem to be covered by the phrase "set of all integers". What do you think?

Comment (rowlesmr) [reply ddl.dic:1732]: That sounds good to me.

"`A number from the set of all integers.`"

**Review by vaitkus — APPROVED** [review]

**Pull request merged**